src/index.js

```diff
@@ -58,16 +59,30 @@ const unvisit = (input, schema, getDenormalizedEntity) => {
     return method(schema, input, unvisit, getDenormalizedEntity);
   }
 
+  if (input === undefined || input === null) {
```

Comment:
Another related thing I just realized: this currently throws that same unexpected error if an entity is not found:

```
const mySchema = new schema.Entity('tacos');
const entities = {
  tacos: {
    1: { id: 1, type: 'foo' }
  }
};

> denormalize(3, mySchema, entities)
TypeError: Cannot read property 'id' of undefined
    at EntitySchema._getId (~/normalizr/dist/normalizr.js:226:65)
    at EntitySchema.getId (~/normalizr/dist/normalizr.js:245:19)
    at ~/normalizr/dist/normalizr.js:635:21
    at EntitySchema.denormalize (~/normalizr/dist/normalizr.js:273:20)
    at unvisit (~/normalizr/dist/normalizr.js:612:17)
    at Object.denormalize$1 [as denormalize] (~/normalizr/dist/normalizr.js:651:10)
    at repl:1:3
    at realRunInThisContextScript (vm.js:22:35)
    at sigintHandlersWrap (vm.js:98:12)
    at ContextifyScript.Script.runInThisContext (vm.js:24:12)
```

I kinda feel like that should just return `undefined`. In my apps, I think I'd rather just ignore missing data, especially when denormalizing nested data.

I could also see it being configurable so you can handle how you want (e.g. initialize with empty object, return null,undefined, throw error)
```
denormalize(3, mySchema, entities, { onNotFound: (id, schema) => undefined })
denormalize(3, mySchema, entities, { onNotFound: (id, schema) => { id } })
denormalize(3, mySchema, entities, { onNotFound: (id, schema) => throw new Error(`could not find ${schema.name}` })
```

Thoughts? If you agree I'd probably remove this line and just handle that below if an entity is not found in the store.